tests/inject/karma.conf.js

```diff
@@ -6,22 +9,40 @@ const rollupPluginReplace = require('@rollup/plugin-replace');
 const rollupPluginTypescript = require('@rollup/plugin-typescript');
 const typescript = require('typescript');
 const {getTestDestDir, rootPath} = require('../../tasks/paths');
+const karmaPluginEchoServer = require('./support/echo-server');
 
 /**
- * @param   {import('karma').Config} config
- * @returns {import('karma').ConfigOptions}
+ * @param {LocalConfig} config
+ * @param {Record<string, string>} env
+ * @returns {ConfigOptions}
  */
-function configureKarma(config) {
-    const headless = config.headless || process.env.KARMA_HEADLESS || false;
+function configureKarma(config, env) {
```

Comment:
I set up a test for this method, so I parameterized the `process.env` global here.